src/odemis/gui/main.xrc

```diff
@@ -5029,6 +5029,24 @@
                       <border>5</border>
                     </object>
                     <orient>wxVERTICAL</orient>
+                    <object class="sizeritem">
```

Comment:
This new widget is part of the plugin window. Any plugin can use it to display some text above the buttons. Let's name it in generic way. Maybe `lbl_acquisition_info`.